src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -133,15 +133,20 @@ public function editAction(Request $request, int $productId): Response
      */
     private function renderProductForm(FormInterface $productForm, ?int $productId = null): Response
     {
+        $statsLink = null;
+        $productForEditing = null;
```

Comment:
```suggestion
        $statsLink = $productForEditing = null;
```

Reply:
Indeed `productForEditing` doesn't seem to be used